src/handle_init_contract.c

```diff
@@ -1,5 +1,4 @@
 #include "plugin.h"
-#include "utils.h"
```

Comment:
Removing this results in a compilation warning:
```
src/handle_init_contract.c:32:10: warning: implicit declaration of function 'find_selector' is invalid in C99 [-Wimplicit-function-declaration]
    if (!find_selector(U4BE(msg->selector, 0), SELECTORS, SELECTOR_COUNT, &index)) {
         ^
```

This file has been renamed in the Ethereum Plugin SDK, so we should change this line to:
```c
#include "plugin_utils.h"
```

Reply:
Thanks for catching that–updated!